README.md

```diff
@@ -122,6 +125,23 @@ tracing_span->endSpan();
 tracer->report(std::move(tracing_context));
 ```
 
+#### Skywalking CDS
```

Comment:
After this PR merged or released, I think you need to reword the `Value Format Example` of `configuration-discovery.default.agentConfigurations` here, https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/dynamic-config.md

Change to something like, 
```
[Java agent](../service-agent/java-agent/configuration-discovery.md), [cpp2sky SDK](your link)
```